homework8/task3.py

```diff
@@ -0,0 +1,16 @@
+class EqualSignContext:
+    def __init__(self):
+        self.sign = "="
+
+    def __enter__(self):
+        print(self.sign*10)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print(self.sign*10)
```

Comment:
Move this line to the end of the method. And put the return statement after (move it outside the `if` block